ports/esp32/boards/ESP32_QEMU_C3/board.md

```diff
@@ -0,0 +1,7 @@
+The following files are firmware images that should work on most
+ESP32-C3-based boards with 4MiB of flash, including WROOM and MINI modules,
+that use the revision 3 silicon (or newer).
+
+USB serial/JTAG support is enabled on pin 18 and 19. Note that this
+is not a full USB stack, the C3 just provides a CDC/ACM class serial
+and JTAG interface.
```

Comment:
So, this board also works on hardware as well as qemu?  Won't the ETH break on hardware?

Reply:
We can make it a variant of esp32 generic c3 instead.

Reply:
OPENETH usage will break on hardware, other than that the build is compatible with both.

Reply:
I've marked as un-resolved to keep track of this:  either this should be a board variant, or this text should be changed to describe what this "board" is rather than being copy-paste from the C3 board.

Reply:
Changed to board variant